src/main/java/duke/error/ImportTaskException.java

```diff
@@ -0,0 +1,7 @@
+package duke.error;
+
+/**
+ * Throws if task cannot be imported properly. 
```

Comment:
probably an incorrect javadoc comment. This is the exception that gets thrown, it doesn't throw anything as such